dlp/src/test/java/dlp/snippets/TemplatesTests.java

```diff
@@ -88,6 +92,24 @@ public void setUp() {
     System.setOut(new PrintStream(bout));
   }
 
+  @AfterClass
+  public static void cleanupTemplates() throws IOException {
+    try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
+      ListInspectTemplatesRequest listInspectTemplatesRequest =
+          ListInspectTemplatesRequest.newBuilder()
+              .setParent(ProjectName.of(PROJECT_ID).toString())
+              .setPageSize(1)
+              .build();
+      ListInspectTemplatesPagedResponse response = dlpServiceClient
+          .listInspectTemplates(listInspectTemplatesRequest);
+      for (InspectTemplate template : response.getPage().getResponse().getInspectTemplatesList()) {
+        DeleteInspectTemplateRequest deleteInspectTemplateRequest =
+            DeleteInspectTemplateRequest.newBuilder().setName(template.getName()).build();
+        dlpServiceClient.deleteInspectTemplate(deleteInspectTemplateRequest);
+      }
+    }
+  }
```

Comment:
This might be a bit too aggressive as your deleting all existing templates, some might actually be in use by another test running at the same time.

Reply:
changed in latest commit